packages/cloud_firestore/example/android/gradle.properties

```diff
@@ -1 +1,2 @@
 org.gradle.jvmargs=-Xmx1536M
+android.enableR8=true
```

Comment:
Do we need this?

Reply:
Sorry, I missed this. It seems that this automatically added by Android Studio.